lib/devcycle-ruby-server-sdk/api/devcycle_api.rb

```diff
@@ -15,9 +15,13 @@
 module DevCycle
   class DVCClient
     attr_accessor :api_client
+    attr_accessor :localbucketing
```

Comment:
This is only exposed here for use by the test in `examples/sinatra/testing.rb` right?

Reply:
yep